docs/content/configuration/creating_pvc.md

```diff
@@ -4,9 +4,11 @@ Create a PersistentVolumeClaim (PVC) YAML file for a persistent volume (PV).
 
 The IBM® block storage CSI driver supports using both file system and raw block volume modes.
 
-**Important:** If not defined, the default mode is `Filesystem`. Be sure to define the mode as `Block` if this configuration is preferred.
+**Important:**
+  - If not defined, the default mode is `Filesystem`. Be sure to define the mode as `Block` if this configuration is preferred.
+  - The volume group labels are not pre-defined. Be sure to match the selector in the target volume group (`spec.source.selector`). For an example of creating a PVC using the VolumeGroup configuration, see [Creating a PVC in a volume group](#creating-a-pvc-in-a-volume-group).
 
-**Note:** The examples below create the PVC with a storage size 1 Gb. This can be changed, per customer needs.
+**Note:** The examples below create the PVC with a storage size 1 Gb and using the dynamic volume group feature. This can be changed, per customer needs.
```

Comment:
not all using the dynamic volume group feature

Reply:
updated